src/blocks/Navbar.tsx

```diff
@@ -1,14 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Flex } from 'rebass';
-import LogoIcon from '../icons/LogoIcon';
+import { space, SpaceProps } from 'styled-system';
+import { Drawer } from '@material-ui/core';
+import { useWindowSize } from 'use-window-size-hook';
+
 import { Bold, Medium, Regular } from '../ui/Text';
 import styled from 'styled-components';
-import { space } from 'styled-system';
+import media from 'styled-media-query';
+
+import MenuButton from '../buttons/MenuButton';
+import LogoIcon from '../icons/LogoIcon';
 
-const MenuText = styled.a`
+const MenuText = styled.a<SpaceProps>`
   font-weight: 400;
-  color: #4F5665;
+  color: #4f5665;
```

Comment:
do we have a lint to fix this color case?

Reply:
We can apply stylelint to Styled Components and then apply the ``color-hex-case`` rule 

Refer to:
https://github.com/styled-components/stylelint-processor-styled-components
https://stylelint.io/user-guide/rules/color-hex-case

Reply:
this is great, I've forgot about stylelint

we use it on our internal projects at @entria